txstratum/api.py

```diff
@@ -86,8 +86,22 @@ def __init__(
         self.app.router.add_post("/submit-job", self.submit_job)
         self.app.router.add_post("/cancel-job", self.cancel_job)
 
+        # Signal handler to add CORS headers when preparing the response
+        self.app.on_response_prepare.append(self.on_prepare)
+
         self.fix_invalid_timestamp: bool = fix_invalid_timestamp
 
+    async def on_prepare(
+        self, request: web.Request, response: web.StreamResponse
+    ) -> None:
+        """Set CORS headers for all responses on_prepare."""
+        response.headers["Access-Control-Allow-Origin"] = "http://localhost:3000"
```

Comment:
Should we add a comment explaning the reason why localhost is being allowed?

Reply:
https://github.com/HathorNetwork/tx-mining-service/pull/116/commits/7ae63b92d9d36fca5aaa059fcd8fc4334803b740